examples/gadgets/operators/otel_metrics/README.md

```diff
@@ -0,0 +1,19 @@
+# Open Telemetry Metrics Exporter Operator
+
+This example shows how to export metrics using Open Telemetry / Prometheus.
+
+### How to run
+
+```bash
+$ go run -exec sudo .
+```
+
+### What it does
+
+<!-- markdown-link-check-disable-next-line -->
```

Comment:
It checked the url in the next line - which is just a demo and invalid.

Reply:
I didn't check - this is however how we use it several times in our documentation - whenever we have an example URL, we use that indicator to prevent the check.